ui/tests/acceptance/aws-test.js

```diff
@@ -31,39 +37,37 @@ module('Acceptance | aws secret backend', function (hooks) {
     await click('[data-test-secret-backend-configure]');
 
     assert.strictEqual(currentURL(), `/vault/settings/secrets/configure/${path}`);
-    assert.ok(findAll('[data-test-aws-root-creds-form]').length, 'renders the empty root creds form');
-    assert.ok(findAll('[data-test-aws-link="root-creds"]').length, 'renders the root creds link');
-    assert.ok(findAll('[data-test-aws-link="leases"]').length, 'renders the leases config link');
+
+    assert.dom('[data-test-aws-root-creds-form]').exists('renders the empty root creds form');
+    assert.dom(GENERAL.tab('access-to-aws')).exists('renders the root creds tab');
+    assert.dom(GENERAL.tab('lease')).exists('renders the leases config tab');
 
     await fillIn('[data-test-aws-input="accessKey"]', 'foo');
     await fillIn('[data-test-aws-input="secretKey"]', 'bar');
 
     await click('[data-test-aws-input="root-save"]');
 
-    assert.ok(
-      find('[data-test-flash-message]').textContent.trim(),
-      `The backend configuration saved successfully!`
+    assert.true(
+      this.flashSuccessSpy.calledWith('The backend configuration saved successfully!'),
+      'success flash message is rendered'
     );
 
-    await click('[data-test-aws-link="leases"]');
+    await click(GENERAL.tab('lease'));
 
     await click('[data-test-aws-input="lease-save"]');
 
-    assert.ok(
-      find('[data-test-flash-message]').textContent.trim(),
-      `The backend configuration saved successfully!`
+    assert.true(
```

Comment:
the previous assertions would've passed even if the same flash message rendered, so this guarantees that a fresh message is rendered.